<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>report</class>
 <widget class="QMainWindow" name="report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relatório De Saltos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>← Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Relatório De Saltos Por Mês</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(68, 165, 255)</string>
    </property>
    <property name="text">
     <string>FILTRAR</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="firstDate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QDateEdit" name="lastDate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SELECIONE UM INTERVALO DE MESES</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="monthTable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>500</width>
      <height>191</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>249</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <column>
     <property name="text">
      <string>MESES</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NÚMERO DE SALTOS</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="averageLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MÉDIA DE SALTOS DO INTERVALO</string>
    </property>
   </widget>
   <widget class="QLabel" name="averageCountLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>500</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
